packages/back-end/src/models/ExperimentSnapshotModel.ts

```diff
@@ -502,13 +504,10 @@ export async function getLatestSnapshotMultipleExperiments(
 
 export async function createExperimentSnapshotModel({
   data,
-  context,
 }: {
   data: ExperimentSnapshotInterface;
-  context: Context;
 }): Promise<ExperimentSnapshotInterface> {
   const created = await ExperimentSnapshotModel.create(data);
-  await notifyExperimentChange({ context, snapshot: created });
```

Comment:
Does this mean that we won't send notification for synchronous changes where the snapshot is created in a final format? (Unsure if we even have this though, maybe for manual snapshots?)

Reply:
I don't think we ever do this in practice. I could probably leave this here because no statuses would change so it would be unlikely to trigger any kind of notification, but I think we should only have one place where the "state of the experiment" is evaluated, including updating the experiment `analysisSummary` and firing any notifications. Having this here is a bit of an odd-man out. If we find out we're missing these notifications, maybe we should also be updating `analysisSummary`?

Reply:
That's an interesting point -- agreed that we likely need to refactor some stuff to ensure we have a single place where 'new snapshot was created (regardless of how), do what's needed'. Can do that on a later PR.

For `analysisSummary` at the start we required the health query, therefor it would always be async via the QueryRunner.

Reply:
I'm not sure I follow, but do you think I should just add this back in here?

Reply:
No need to add it back.